lldb/source/Utility/ArchSpec.cpp

```diff
@@ -617,6 +617,14 @@ void ArchSpec::SetFlags(const std::string &elf_abi) {
   SetFlags(flag);
 }
 
+void ArchSpec::SetAdditionalDisassemblyFeatureStr(
```

Comment:
I'm undecided on this, it's a style question.  This method takes a feature to add to the list of features we're enabling.  The caller must have a comma after the name, e.g. `+xqci,` that we see in `ObjectFileELF::ParseRISCVAttributes`.  Would it be a more natural API if the caller only sends `+xqci` and this method adds the `,` between elements if the `m_additional_disassembly_feature_str` is already non-zero?